backstop/scenarios/side-navigation.js

```diff
@@ -9,18 +9,18 @@ module.exports = [
     actions: [click('#test-expand')],
     selectors: ['#demo-side-navigation'],
   }),
+  scenario('Type submenu', {
+    actions: [click('#myDocumentsButton'), hover('#demo-take-hover')],
+    selectors: ['#demo-side-navigation'],
+  }),
 
   // Hover & focus states
   scenario('State collapsed hover and focus', {
     actions: [hover('#test-tab-2'), focus('#test-tab-1')],
     selectors: ['#demo-side-navigation'],
   }),
   scenario('State expanded hover and focus', {
-    actions: [
-      click('#test-expand'),
-      hover('#test-tab-2'),
-      focus('#test-tab-1'),
-    ],
+    actions: [click('#test-expand'), hover('#test-tab-2'), focus('#test-tab-1')],
```

Comment:
Is focus visible on dark theme? Because I can't see it.

Reply:
Hmm… I am noticing that now too in both light and dark theme.  The non-expanded side navigation tests are working correctly tho.  Is it possible the `click` action is being applied last so it is stealing focus away from the calendar button?

Reply:
Yeah, I see that there is no focus. I am looking into that.

Reply:
Made a fix https://github.com/iTwin/iTwinUI/pull/295

Reply:
Any chance that PR might have caused this change? Looks like old one had hover effect leftover in expander.
![image](https://user-images.githubusercontent.com/9084735/134245898-b7d5d1e9-f44b-4ff2-9529-2b0abc13ef79.png)

This is the only change I can see in all test images currently.

Reply:
I think so. Previously `click` would leave hover on the clicked element but sometimes it wouldn't leave hover effect.